src/modules/ws/websocket.ts

```diff
@@ -0,0 +1,48 @@
+import SendMessageValidator from './message_validator';
+
+interface WebSocketConnection {
+  ws: WebSocket;
+  sendMessage(key: string, data: Record<string, unknown>): void;
+}
+
+const WEBSOCKET_URL = process.env.REACT_APP_WEBSOCKETURL || '';
+
+/**
+ * Interface that the client can use for Websocket Connection.
+ */
+class WebSocketConnection {
+  public constructor(url: string = WEBSOCKET_URL) {
+    this.ws = new WebSocket(url);
+    this.ws.onopen = (event) => {
+      console.log(event);
+      this.ws.send('');
+    };
+
+    this.ws.onclose = (event) => {
+      console.log(event);
+    };
+
+    this.ws.onerror = (error) => {
+      console.log(error);
```

Comment:
suggest we build a common logger. has info related to given build

Reply:
maybe trello todo
or leave todo line

Reply:
Okies